components/brave_ads/README.md

```diff
@@ -6,29 +6,36 @@ Users earn tokens by viewing privacy-first Brave Ads. Ads presented are based on
-| Creative set  | A creative set refers to a collection of related creative instances or variations of an advertisement  |
-| Day parting  | Day parting is a strategy that involves scheduling and targeting ads based on specific times of the day to maximize their effectiveness  |
-| Delivery  | Delivering ad content to the user, also known as serving  |
-| Frequency capping  | Limits how many times a given ad will be shown within a specified period, also known as exclusion rules  |
-| Impression  | An impression is when an ad is displayed or shown to a user, also known as a view  |
-| Multi-armed bandits  | Multi-armed bandits optimize the allocation of resources between different ad variations or strategies to maximize performance  |
-| Pacing  | Rate at which an ad campaign uses up its pre-set number of impressions  |
-| Placement  | The location at which an ad is displayed to the user  |
-| Priority  | Priority at which an ad campaign uses up its impressions  |
-| Purchase intent  | Likelihood or inclination of a consumer to make a purchase based on their expressed interest or behavior. It indicates the consumer's readiness to buy a product or service  |
-| Split testing  | Comparing two ad or landing page versions to determine which performs better, also known as A/B testing  |
-| Targeting  | Delivering an ad to the appropriate audience through behavioral, contextual, or geographic targeting  |
-| Text classification  | Target audiences based on the analysis and understanding of textual data to deliver more relevant and personalized advertisements  |
-| Text embedding  | Converts text-based information, such as user reviews or product descriptions, into numerical representations (embeddings), enabling more precise audience targeting based on the semantic meaning of the text  |
-| Transfer  | Delivery of an advertiser's website, i.e., the landing page after the user clicks on an ad, also known as landed  |
-| User attention  | Measure user attention to drive performance  |
+| Ad event  | Ad events are actions or interactions with an advertisement, such as a user viewing or clicking an ad.  |
+| Advertiser  | An advertiser is someone or a company that promotes products, services, or ideas through various mediums to reach and persuade potential customers or target audiences.  |
+| Anti-targeting  | Exclude users from receiving targeted advertisements based on specific attributes such as demographics, interests, or behaviors.  |
+| Behavioral  | Behavioral advertising targets ads based on the user's previous online behavior, such as browsing history and interactions, to deliver relevant and personalized advertisements.  |
+| Campaign  | A campaign is a coordinated series of marketing activities and messages designed to achieve specific goals within a defined timeframe.  |
+| Catalog  | Inventory of available advertisements.  |
```

Comment:
`inventory` is generally used to describe the supply, in this case, the catalog describes the demand

Reply:
I can, but I decided to remove Ads from other terms because this file is only for Ads. Your thoughts?

Reply:
Fixed, thanks